composeApp/src/commonMain/sqldelight/org/ooni/probe/data/Measurement.sq

```diff
@@ -62,6 +62,14 @@ SELECT * FROM Measurement
 LEFT JOIN Url ON Measurement.url_id = Url.id
 WHERE Measurement.result_id = ?;
 
+countByResultId:
+SELECT
+    COUNT(*) AS total_measurements,
+    SUM(CASE WHEN is_rerun = 0 AND is_done = 1 AND is_failed = 0 AND is_anomaly = 0 THEN 1 ELSE 0 END) AS ok_measurements,
+    SUM(CASE WHEN is_rerun = 0 AND is_done = 1 AND is_failed = 1 THEN 1 ELSE 0 END) AS failed_measurements
+FROM Measurement
+WHERE result_id = ?;
```

Comment:
Do we need the extra DB query? Can't we just get this values from the whole list?